server/routes/createGoal/createGoalController.test.ts

```diff
@@ -57,6 +56,7 @@ describe('createGoalController', () => {
         'targetDate-year': '2024',
         action: 'submit-form',
       }
+      req.session.addStepForms = new Array<AddStepForm>()
```

Comment:
When you new up a new array here and on L86 and L107; I think you can use the shorter syntax:
```
req.session.addStepForms = []
```